flax/nnx/module.py

```diff
@@ -268,84 +269,26 @@ def perturb(
 
   def iter_modules(self) -> tp.Iterator[tuple[PathParts, Module]]:
     """Recursively iterates over all nested :class:`Module`'s of the current Module, including
-    the current Module.
-
-    ``iter_modules`` creates a generator that yields the path and the Module instance, where
-    the path is a tuple of strings or integers representing the path to the Module from the
-    root Module.
-
-    Example::
-
-      >>> from flax import nnx
-      ...
-      >>> class SubModule(nnx.Module):
-      ...   def __init__(self, din, dout, rngs):
-      ...     self.linear1 = nnx.Linear(din, dout, rngs=rngs)
-      ...     self.linear2 = nnx.Linear(din, dout, rngs=rngs)
-      ...
-      >>> class Block(nnx.Module):
-      ...   def __init__(self, din, dout, *, rngs: nnx.Rngs):
-      ...     self.linear = nnx.Linear(din, dout, rngs=rngs)
-      ...     self.submodule = SubModule(din, dout, rngs=rngs)
-      ...     self.dropout = nnx.Dropout(0.5)
-      ...     self.batch_norm = nnx.BatchNorm(10, rngs=rngs)
-      ...
-      >>> model = Block(2, 5, rngs=nnx.Rngs(0))
-      >>> for path, module in model.iter_modules():
-      ...   print(path, type(module).__name__)
-      ...
-      ('batch_norm',) BatchNorm
-      ('dropout',) Dropout
-      ('linear',) Linear
-      ('submodule', 'linear1') Linear
-      ('submodule', 'linear2') Linear
-      ('submodule',) SubModule
-      () Block
+    the current Module. Alias of ``iter_modules``.
```

Comment:
```suggestion
    the current Module. Alias of ``nnx.iter_modules``.
```

Put a deprecation message.